src/sagas/systemIntakeSaga.ts

```diff
@@ -4,19 +4,21 @@ import { PUT_SYSTEM_INTAKE, GET_SYSTEM_INTAKE } from 'constants/actions';
 
 import { PutSystemIntakeAction } from 'types/systemIntake';
 import { prepareSystemIntakeForApi } from 'data/systemIntake';
-import { storeSystemIntake } from '../actions/systemIntakeActions';
+import { storeSystemIntake } from 'actions/systemIntakeActions';
 
-function putSystemIntakeRequest({ id, formData }: PutSystemIntakeAction) {
+function putSystemIntakeRequest({ formData }: PutSystemIntakeAction) {
   // Make API save request
-  const data = prepareSystemIntakeForApi(id, formData);
+  const data = prepareSystemIntakeForApi(formData);
   return axios.put(`${process.env.REACT_APP_API_ADDRESS}/system_intake`, data);
 }
 
 export function* putSystemIntake(payload: PutSystemIntakeAction) {
   try {
     const response = yield call(putSystemIntakeRequest, payload);
+    // eslint-disable-next-line no-console
```

Comment:
should we be doing something else in here? I know you are only adding the disable lines, but maybe we should be sending these as application logs to somewhere so we can have metrics about things.

Reply:
Maybe? We already log any response on the server, so we might be okay just deleting these console.log()'s